run.sh

```diff
@@ -1,31 +1,38 @@
 #!/bin/sh
 
+#add environmental variable of .env to the bash script
+. ./.env
+
+#Define absolute path for the host and the serv from the relative common path
+HOST_LOG_FILE=$PWD${LOG_FILE}
+SERV_LOG_FILE=/opt/iscsc.fr-notify-bot${LOG_FILE}
```

Comment:
```suggestion
SERV_LOG_FILE=/opt/iscsc.fr-notify-bot/${LOG_FILE}
```
is better because even if `LOG_FILE=/something`
it gives `SERV_LOG_FILE=/opt/iscsc.fr-notify-bot//something`
it doesn't matter: double slashes are ignored

Reply:
also I'd prefer to name it `CONT_LOG_FILE` because `SERV_` isn't really what it is...

Reply:
removed in 52c2335

Reply:
I like this tks!